assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -125,6 +127,159 @@ export default {
         });
       });
     });
+
+    withPluginApi("1.3.0", (api) => {
+      api.addSidebarSection(
+        (BaseCustomSidebarSection, BaseCustomSidebarSectionLink) => {
+          const SidebarChatSectionLink = class extends BaseCustomSidebarSectionLink {
+            constructor({ channel }) {
+              super(...arguments);
+              this.channel = channel;
+            }
+
+            get name() {
+              return this.channel.chatable_id;
```

Comment:
I would recommend a combination of `chatable_type` and `chatable_id` here to properly differentiate between the links.